cms/envs/common.py

```diff
@@ -226,6 +226,7 @@
 PROJECT_ROOT = path(__file__).abspath().dirname().dirname()  # /edx-platform/cms
 REPO_ROOT = PROJECT_ROOT.dirname()
 COMMON_ROOT = REPO_ROOT / "common"
+OPENEDX_ROOT = REPO_ROOT / "openedx"
```

Comment:
@nasthagiri Interesting idea, but I don't think so. We might need to refer to tests or other directories not under core.